packages/core/src/browser/quick-input/quick-input-service.spec.ts

```diff
@@ -0,0 +1,175 @@
+/********************************************************************************
+ * Copyright (C) 2021 Ericsson and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+import * as chai from 'chai';
+import { filterItems, findMatches, QuickPickItem } from './quick-input-service';
+
+const expect = chai.expect;
+
+describe('quick-input-service', () => {
+
+    describe('#findMatches', () => {
+
+        it('should return the proper highlights when matches are found', () => {
+            expect(findMatches('abc', 'a')).deep.equal([{ start: 0, end: 1 }]);
+            expect(findMatches('abc', 'ab')).deep.equal([{ start: 0, end: 1 }, { start: 1, end: 2 }]);
+            expect(findMatches('abc', 'abc')).deep.equal([{ start: 0, end: 1 }, { start: 1, end: 2 }, { start: 2, end: 3 }]);
```

Comment:
'ac' might be a more interesting third case, rather than another one that matches consecutively?